Library/Homebrew/utils/ast.rb

```diff
@@ -190,6 +190,19 @@ def add_stanza(name, value, type: nil)
         tree_rewriter.insert_after(preceding_expr, "\n#{stanza_text(name, value, indent: 2)}")
       end
 
+      sig { returns(T::Boolean) }
+      def include_runtime_cpu_detection?
+        install_node = children.find do |child|
+          (child.is_a? RuboCop::AST::DefNode) && child.method_name == :install
+        end
+
+        return false if install_node.blank?
+
+        install_node.each_node.any? do |node|
+          node.send_type? && node.receiver.const_name == "ENV" && node.method_name == :runtime_cpu_detection
```

Comment:
```suggestion
          node.send_type? && node.receiver&.const_name == "ENV" && node.method_name == :runtime_cpu_detection
```

Keep it for `receiver`, since you can have a send node without a receiver. Looks good otherwise.